ui/StatusQ/src/StatusQ/Core/Utils/Utils.qml

```diff
@@ -336,4 +336,22 @@ QtObject {
         
         return content.length > 0 ? (content + tidSuffix) : tidSuffix
     }
+
+    function htmlToPlainText(html) {
+      if (!html) return "";
+
+      return String(html)
+        .replace(/<br\s*\/?>/gi, "\n")   // line breaks
+        .replace(/<[^>]+>/g, "")         // strip tags
+        .replace(/&nbsp;/g, " ")         // decode a few common entities
+        .replace(/&amp;/g, "&")
+        .replace(/&lt;/g, "<")
+        .replace(/&gt;/g, ">")
+        .trim();
+    }
```

Comment:
Is it intentional to create new js function for that? We already have `StringUtils::plainText` method doing that job, probably in much faster way.